bluesky/run_engine.py

```diff
@@ -268,6 +270,16 @@ def __init__(self, md=None, *, loop=None, md_validator=None):
         self.loop.call_soon(self._check_for_signals)
 
     @property
+    def rewindable(self):
+        return self._rewindable_flag
+
+    @rewindable.setter
+    def rewindable(self, v):
+        self._rewindable_flag = bool(v)
```

Comment:
If I set `rewindable = True` twice (accidentally or otherwise), is it desirable to reset in this situation?